src/control/jobs/CustomExportJob.h

```diff
@@ -17,6 +17,15 @@
 
 #include <PageRange.h>
 
+class Document;
+class PageType;
+
+const string EXPORT_PDF = "PDF files";
```

Comment:
This strings needs to be translatable => _()
Static seems to be not working, so it should be intialized in the constructor

Reply:
Oh but I did that :) 
here:
`addFileFilterToDialog(_C(EXPORT_PDF), "*.pdf");`

Reply:
This will not work. Then remove the constants, and add the Text directly, gettext will not resolve constants.

Reply:
The translation will work, but the source Text will not appear in the Translation editor. Thats what I mean